### PR TITLE
Add upvotes property to LoggedInUser

### PR DESCRIPTION
**Pull request by tiembo**
Includes type converter for storing/retrieving upvotes from database.

### CODE REVIEW

File: core/src/main/java/io/plaidapp/core/data/Converters.kt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+package io.plaidapp.core.data
+
+import android.arch.persistence.room.TypeConverter
+
+/**
+ * Type converters to allow Room to reference complex data types.
+ */
+class Converters {
+    private val delimiter = ","
```
Comment (nickbutcher) [inline core/src/main/java/io/plaidapp/core/data/Converters.kt]: `const`

File: core/src/main/java/io/plaidapp/core/designernews/data/users/model/LoggedInUser.kt [hunk 2]
```diff
@@ -42,5 +42,9 @@ data class LoggedInUser(
 
     @ColumnInfo(name = "potrait_url")
     @SerializedName("portrait_url")
-    val portraitUrl: String? = null
+    val portraitUrl: String? = null,
+
+    @ColumnInfo(name = "upvotes")
+    @SerializedName("upvotes")
+    val upvotes: List<String>
```
Comment (nickbutcher) [inline core/src/main/java/io/plaidapp/core/designernews/data/users/model/LoggedInUser.kt]: Elsewhere we're converting IDs to `Long`s. Should we also do this here?

Comment (florina-muntenescu) [reply core/src/main/java/io/plaidapp/core/designernews/data/users/model/LoggedInUser.kt]: +1 for `Long`

File: core/src/main/java/io/plaidapp/core/data/Converters.kt (hunk 1 above)
Comment (tiembo) [reply core/src/main/java/io/plaidapp/core/data/Converters.kt]: Done

File: core/src/main/java/io/plaidapp/core/designernews/data/users/model/LoggedInUser.kt (hunk 2 above)
Comment (tiembo) [reply core/src/main/java/io/plaidapp/core/designernews/data/users/model/LoggedInUser.kt]: Done

**Pull request merged**